src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
 // write code here
+document.addEventListener('DOMContentLoaded', function () {
+  const listItems = document.querySelectorAll('.tree li');
```

Comment:
Consider adding a check to see if the `listItems` is not empty before proceeding with the `forEach` loop. This can help prevent unnecessary looping in case the `listItems` is empty.